libs/datamodel/core/src/transform/ast_to_dml/db/relations.rs

```diff
@@ -2,51 +2,55 @@ use super::{context::Context, types::RelationField};
 use crate::ast;
 use std::collections::BTreeSet;
 
-#[derive(PartialOrd, Ord, PartialEq, Eq, Debug, Clone)]
+#[derive(PartialOrd, Ord, PartialEq, Eq, Debug)]
```

Comment:
We don't want to clone/copy this?

Reply:
Nope, it's internal. Should not leave the module, we want walkers for the public API.